services/wallet/api.go

```diff
@@ -428,7 +428,11 @@ func (api *API) DeleteEthereumChain(ctx context.Context, chainID uint64) error {
 
 func (api *API) GetEthereumChains(ctx context.Context) ([]*network.CombinedNetwork, error) {
 	log.Debug("call to GetEthereumChains")
-	return api.s.rpcClient.NetworkManager.GetCombinedNetworks()
+	isSepoliaEnabled, err := api.s.accountsDB.GetIsSepoliaEnabled()
```

Comment:
@alaibe this is not needed here, you have access to `nm.accountsDB` from the Manager and that can be resolved there.

Reply:
Actually, that's what you're doing, seems you just forgot to remove `isSepoliaEnabled` param from the `GetCombinedNetworks` function declaration, since it's not used there.

Reply:
indeed 🤦